### PR TITLE
Update whats-new-in-defender-for-office-365.md

### PR DESCRIPTION
**Pull request by VipulMSFT**
Updating the doc to cover Email entity page, Management API updates and Threat Analytics

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 1]
```diff
@@ -38,6 +38,13 @@ Learn more by watching [this video](https://www.youtube.com/watch?v=Tdz6KfruDGo&
 > [!TIP]
 > Don't have Microsoft Defender for Office 365 yet? [Contact sales to start a trial](https://info.microsoft.com/ww-landing-M365SMB-web-contact.html).
 
+## April/May 2021 
+
+- [Email entity page](mdo-email-entity-page.md): A unified 360-degree view of an email with enriched information around Threats, authentication and detections, detonation details, and a brand-new Email Preview experience
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
- [Email entity page](mdo-email-entity-page.md): A unified 360-degree view of an email with enriched information around Threats, authentication and detections, detonation details, and a brand-new Email Preview experience.
```

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 2]
```diff
@@ -38,6 +38,13 @@ Learn more by watching [this video](https://www.youtube.com/watch?v=Tdz6KfruDGo&
...
+- Office 365 Management API: Updates to EmailEvents (RecordType 28) to add Delivery Action, Original and Latest Delivery Locations, and updated Detection details. https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#email-message-events
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
- [Office 365 Management API](/office/office-365-management-api/office-365-management-activity-api-schema#email-message-events): Updates to EmailEvents (RecordType 28) to add Delivery Action, Original and Latest Delivery Locations, and updated Detection details.
```

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 3]
```diff
@@ -38,6 +38,13 @@ Learn more by watching [this video](https://www.youtube.com/watch?v=Tdz6KfruDGo&
 > [!TIP]
 > Don't have Microsoft Defender for Office 365 yet? [Contact sales to start a trial](https://info.microsoft.com/ww-landing-M365SMB-web-contact.html).
 
+## April/May 2021 
+
+- [Email entity page](mdo-email-entity-page.md): A unified 360-degree view of an email with enriched information around Threats, authentication and detections, detonation details, and a brand-new Email Preview experience
+- Office 365 Management API: Updates to EmailEvents (RecordType 28) to add Delivery Action, Original and Latest Delivery Locations, and updated Detection details. https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#email-message-events 
+- Threat Analytics for Defender for Office 365: View active threat actors, popular techniques and attack surfaces, along with extensive reporting from Microsoft researchers around ongoing campaigns. https://docs.microsoft.com/en-us/microsoft-365/security/defender/threat-analytics
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
- [Threat Analytics for Defender for Office 365](/microsoft-365/security/defender/threat-analytics): View active threat actors, popular techniques and attack surfaces, along with extensive reporting from Microsoft researchers around ongoing campaigns. 
```

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 4]
```diff
@@ -38,6 +38,13 @@ Learn more by watching [this video](https://www.youtube.com/watch?v=Tdz6KfruDGo&
 > [!TIP]
 > Don't have Microsoft Defender for Office 365 yet? [Contact sales to start a trial](https://info.microsoft.com/ww-landing-M365SMB-web-contact.html).
 
+## April/May 2021 
+
+- [Email entity page](mdo-email-entity-page.md): A unified 360-degree view of an email with enriched information around Threats, authentication and detections, detonation details, and a brand-new Email Preview experience
+- Office 365 Management API: Updates to EmailEvents (RecordType 28) to add Delivery Action, Original and Latest Delivery Locations, and updated Detection details. https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#email-message-events 
+- Threat Analytics for Defender for Office 365: View active threat actors, popular techniques and attack surfaces, along with extensive reporting from Microsoft researchers around ongoing campaigns. https://docs.microsoft.com/en-us/microsoft-365/security/defender/threat-analytics 
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @VipulMSFT, please commit my suggested changes if you agree with them. Many thanks.

**Comment by VipulMSFT** [comment]
@JohanFreelancer9 - can you please have a relook at the URLs? I agreed with all the changes, but looks like the URL hyperlink may not be working. 
This includes the Threat Analytics link over here: 
https://docs.microsoft.com/en-us/microsoft-365/security/defender/threat-analytics?view=o365-worldwide 

And Management API link over here: https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#email-message-events

**Review by JohanFreelancer9 — APPROVED** [review]
@VipulMSFT - Thank you for committing my suggested changes. The relative links might not work in the preview version for the PR, but they'll work when the article is merged to master.

**Comment by VipulMSFT** [comment]
@yogkumgit the PR has been approved by Johan

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Pull request merged**